app/src/main/java/com/gdgistanbul/attendence/ui/login/LoginFragment.kt

```diff
@@ -0,0 +1,71 @@
+package com.gdgistanbul.attendence.ui.login
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.navigation.fragment.findNavController
+import com.gdgistanbul.attendence.R
+import com.gdgistanbul.attendence.util.AUTHORIZATION_URL
+import com.gdgistanbul.attendence.util.CODE
+import com.gdgistanbul.attendence.util.REDIRECT_URI
```

Comment:
Where are these(AUTHORIZATION_URL, CODE, REDIRECT_URI) constants are defined? I am not able to find them.

Reply:
It was decided not to commit this constants. You can access from #meetupattendee channel.